### DISCUSSION TITLE
How do I set an include path for pylint to use?

### DISCUSSION THREAD

**Discussion by jackiescanlon** [microsoft/vscode-pylint#407] [Q&A] [answered]
I am using the Pylint extension on VS Code for Ubuntu 22.04.

I have added extra python packages to my python path in `settings.json`. For example I added a path to ROS2 packages.

```json
    "python.autoComplete.extraPaths": [
        "/opt/ros/iron/lib/python3.10/site-packages",
    ],
```

I know that the path is valid and working because Python IntelliSense is able to tell me module and function descriptions from packages that are on the path. However, Pylint does not seem to use the path I have set in `settings.json`, because it gives me an import error.
![image](https://github.com/microsoft/vscode-pylint/assets/43035541/b5776902-3085-4372-b847-87a28f88c599)

How do I tell Pylint to use the include path in `settings.json`? Or, if that's not possible, is there another way to configure an include path for Pylint?

**Comment by karthiknadig** [comment, marked as answer]
Can you change that to `python.analysis.extraPaths`? That should work for both `Pylance` and `Pylint`

**Reply by jackiescanlon** [reply]
That did the trick, thank you so much!
